src-ts/config/constants.ts

```diff
@@ -1,4 +1,4 @@
 export enum ToolTitle {
-    settings = 'Account Settings',
+    settings = 'Profile Settings',
```

Comment:
This change should be reverted -- the Figma comp is out of date. This should say "Account Settings".